src/fxapp/ReportManager.java

```diff
@@ -31,4 +31,14 @@ public void addReport(Report report) {
     public Stream<Report> getReports() {
         return reports.stream();
     }
+
+    public Report get(int i) {
```

Comment:
This method isn't necessary, because getReports already exposes all the reports. It's stupidly verbose in Java but to get the first report (in the other class, where we need it) you could do:
```java
reportManager.getReports().findFirst().get();
```

And I think if you wanted to be safe about it, findFirst() returns like an `Optional` object or something that has a boolean of whether it exists or not. So instead of doing a try catch, since those are ugly, you could do something like
```java
Optional<Report> reportMaybe = reportManager.getReports().findFirst();
Report report = null;
if (reportMaybe.isPresent()) { report = reportMaybe.get() } // idk about the exact syntax but it should be similar
```